jekyll/_cci2/high-availability.md

```diff
@@ -6,7 +6,7 @@ order: 20
 description: "Configuring High Availability for CircleCI 2.0"
 ---
 
-**NOTE: Your CircleCI license must include support for high-availability in order to use this configuration. Please contact your account manager if you have questions.**
+**Note: To use this configuration, your CircleCI license must include support for high availability. Please contact your Customer Success Manager if you'd like to set up a High Availability Installation on CircleCI 2.0.**
```

Comment:
I've purposefully avoided "CSM" in docs, as many people for whom this is relevant don't yet have one and are dealing with AE/SE.

Reply:
Makes sense. But if we block HA configuration with CSMs, we should be explicit about that relationship. @michelle-luna thoughts?